### PR TITLE
feat: upgrade helmfile

### LINKED ISSUE #250: Upgrade tools to use new helmfile
When this one is released: https://github.com/roboll/helmfile/pull/1618

we should upgrade tools to use that helmfile and add:

`disableValidationOnInstall: true`

to our helmfile defaults so that it is able to install crds and the chart in one go upon first install.

### PR DESCRIPTION
**Pull request by rawc0der**
Updated helmfile to version 0.136.0 in `otomi/tools` image.
Closes #250

### CODE REVIEW

**Comment by rawc0der** [comment]
I also pushed to dockerhub using the following tag: `otomi/tools:1.4.13`

**Comment by rawc0der** [comment]
@Morriz, should I remove the updated version in core `Dockerfile`?

**Comment by rawc0der** [comment]
> I also pushed to dockerhub using the following tag: `otomi/tools:1.4.13`

This was not actually correct. 
Maybe we need to use the same tag as the core is now depending on (`otomi/tools:1.4.10`), push that to override the image. Then we can merge this.. 

Let me know if this is ok to change.

**Comment by Morriz** [comment]
Don't overwrite, just update the source to use the new tools image.

**Review by Morriz — CHANGES_REQUESTED** [review]
missing the glue

**Comment by rawc0der** [comment]
ok, so the new version tag becomes `otomi/tools:1.4.10`.

**Review by Morriz — CHANGES_REQUESTED** [review]
I think you missed the part that was mentioned:

`disableValidationOnInstall: true`

File: helmfile.d/snippets/templates.gotmpl [hunk 1]
```diff
@@ -2,23 +2,27 @@ templates:
   default: &default
     missingFileHandler: Debug
     chart: ../charts/{{ .Release.Name }}
+    disableValidationOnInstall: true
```
Comment (Morriz) [inline helmfile.d/snippets/templates.gotmpl:5]: Wrong place. Please put it in the defaults ;)

**Review by Morriz — CHANGES_REQUESTED** [review]

**Comment by Dunky13** [comment]
Discussed with Jeho and Alin to add the jq upgrade to this PR to keep it as 1 release, as the jq upgrade does not add breaking changes.

File: helmfile.d/snippets/templates.gotmpl (hunk 1 above)
Comment (rawc0der) [reply helmfile.d/snippets/templates.gotmpl]: that works too, but I changed it now.
For setting it in the defaults file, the property is called `disableValidation`.
It is fine if we want to have them preset as true for every release, then override it if needed on release level.

File: tools/Dockerfile [hunk 2]
```diff
@@ -41,10 +42,13 @@ RUN apt-get update -qq \
   vim \
   nano \
   groff \
-  jq \
   awscli \
   && rm -rf /var/lib/apt/lists/*
 
+RUN curl -L https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 --output /usr/bin/jq && \
```
Comment (j-zimnowoda) [inline tools/Dockerfile:48]: As we agreed, thanks.

**Review by j-zimnowoda — APPROVED** [review]
Looks good, thanks. After testing please do not forget to publish new otomi-tools version  ;)

**Comment by Morriz** [comment]
I see that the last 2 builds failed @j-zimnowoda and @rawc0der. Did you check why? And is it wise to approve a PR with failing build?

**Comment by rawc0der** [comment]
> I see that the last 2 builds failed @j-zimnowoda and @rawc0der. Did you check why? And is it wise to approve a PR with failing build?

I did not build `otomi/tools:1.4.11` yet. That needs to happen when version is agreed upon and can be build without conflicts. If we are ready to merge I will push that image.

**Comment by rawc0der** [comment]
Ok, image with version `1.4.11` is built and pushed but there is another error regarding `ajv` tool.
```
error: parameter --extend-refs is unknown
usage:
    validate:  ajv [validate] -s schema[.json] -d data[.json]
```
@j-zimnowoda can you please take a look why it is failing now?

**Comment by j-zimnowoda** [comment]
> I see that the last 2 builds failed @j-zimnowoda and @rawc0der. Did you check why? And is it wise to approve a PR with failing build?

Not it is not wise. didn't see that.

File: helmfile.d/snippets/templates.gotmpl (hunk 1 above)
Comment (j-zimnowoda) [reply helmfile.d/snippets/templates.gotmpl]: > Ok, image with version `1.4.11` is built and pushed but there is another error regarding `ajv` tool.
> 
> ```
> error: parameter --extend-refs is unknown
> usage:
>     validate:  ajv [validate] -s schema[.json] -d data[.json]
> ```
> 
> @j-zimnowoda can you please take a look why it is failing now?

On it

Comment (j-zimnowoda) [reply helmfile.d/snippets/templates.gotmpl]: The ajv 7.0.0 has removed `--extend-refs` option. https://openbase.com/js/ajv/versions

> extendRefs: $ref siblings are validated by default (consistent with draft 2019-09), ignoreKeywordsWithRef (deprecated) can be used instead to ignore $ref siblings.

Comment (rawc0der) [reply helmfile.d/snippets/templates.gotmpl]: ok, then I am changing it to install fixed version 3.3.0, this was used before the breaking change.

**Review by Morriz — APPROVED** [review]

**Pull request merged**